Task_2_2a_Cons_Dr_Methods/Box.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+
+namespace Task_2_2a_Cons_Dr_Methods
+{
+    class Box
+    {
+        //private int top;
+        //internal int Left;
+
+        //1.  Implement public  auto-implement properties for start position (point position)
+        //auto-implement properties for width and height of the box
+        //and auto-implement properties for a symbol of a given set of valid characters (*, + ,.) to be used for the border 
+        //and a message inside the box
+
+        public int Width { get; set; } = Convert.ToInt32(Console.ReadLine());
```

Comment:
it would be better to transfer keyboard input logic (Convert.ToInt32(Console.ReadLine());) to  static void Main(string[] args)

Reply:
ok